src/App.tsx

```diff
@@ -1,9 +1,67 @@
+import { useCallback, useEffect, useState } from 'react';
 import './App.scss';
+import {
+  Link, Redirect, Route, Switch,
+} from 'react-router-dom';
+import { PeopleTable } from './components/PeopleTable';
+import { getPeople } from './api';
+import { Person } from './PersonType';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [textError, setTextError] = useState<string>('');
+
+  const getData = useCallback(async () => {
+    try {
+      const gottenPeople = await getPeople();
+
+      setPeople(gottenPeople);
+      setTextError('');
+    } catch {
+      setTextError('Somthing going wrong. Please try again,');
+    }
+  }, [people]);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  return (
+    <div className="App">
+      <header className="App__header">
+        <nav className="navbar">
+          <div className="navbar__list">
+            <Link to="/home" className="navbar__item">
+              Home
+            </Link>
+            <Link to="/people" className="navbar__item">
+              People
+            </Link>
+          </div>
+        </nav>
+      </header>
+
```

Comment:
7. Add a `Header` visible everywhere with navigation links to both pages

I think it meant do Header component